src/bot/commands/whois.js

```diff
@@ -1,89 +1,131 @@
+import createLogger from 'another-logger';
+const log = createLogger({label: 'cmd:whois'});
 import {Command} from 'yuuko';
 import config from '../../../config';
 import {parseUser, formatDate} from '../util/discord';
 import {escape} from '../util/formatting';
 
 // function for generating the reddit info
 async function redditLine (userID, guildID, db) {
-	const results = await db.collection('redditAccounts').find({userID, guildID}).toArray();
-	return `__Reddit accounts: **${results.length}**__${
-		results.map(r => `\n- <https://www.reddit.com/u/${encodeURIComponent(r.redditName)}>`).join('')
-	}`;
+	let message = '__Reddit accounts: N/A__';
```

Comment:
This may be too much of a nitpick, but I feel like we don't need a variable for this. We can just return the value from inside `try` and return the "N/A" string as a literal from within `catch` (or outside the `try`/`catch` entirely, identical behavior if `try` ends with a `return`).

Applies to all these functions that have been changed, but I don't really care that much. There's also merits to setting it up as a variable, easier to modify the code later if we need to add extra logic, so do whatever feels right to you and I'll approve the PR either way.

Reply:
Okay I've removed the unnecessary variable